### PR TITLE
upgrade python-urllib3 python-virtualenv python-wcwidth python-websocket-client

### PR DESCRIPTION
**Pull request by nicogbg**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

upgrade python-urllib3 python-virtualenv python-wcwidth python-websocket-client

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

**Review by PawelWMS — APPROVED** [review]

File: SPECS/python-wcwidth/python-wcwidth.spec [hunk 1]
```diff
@@ -34,14 +45,21 @@ This Library is mainly for those implementing a Terminal Emulator, or programs t
 %py3_install
 
 %check
-%python3 setup.py test
+pip3 install \
```
Comment (mfrw) [inline SPECS/python-wcwidth/python-wcwidth.spec:48]: In my opinion, can we please double check the `%check` section, it seems it returns:

```bash
"====== CHECK DONE python-virtualenv. EXIT STATUS 4"
```

File: SPECS/python-websocket-client/python-websocket-client.spec [hunk 2]
```diff
@@ -46,11 +46,11 @@ pip3 install unittest2==0.8.0
 %defattr(-,root,root,-)
 %license LICENSE
 %{python3_sitelib}/*
-%{_bindir}/wsdump.py
+%{_bindir}/wsdump
 
 %changelog
-* Tue Feb 08 2022 Muhammad Falak <mwani@microsoft.com> - 0.56.0-4
-- Install `six` & `unittest2` to enable ptest.
+* Fri Mar 25 2022 Nicolas Guibourge <nicolasg@microsoft.com> - 1.3.1-1
```
Comment (mfrw) [inline SPECS/python-websocket-client/python-websocket-client.spec:52]: The `%check` section for this also seems to break:

```bash
"====== CHECK DONE python-websocket-client. EXIT STATUS 1"
```

**Review by mfrw — CHANGES_REQUESTED** [review]
A few clarifications regarding the `%check` section :)

**Review by mfrw — APPROVED** [review]
LGTM :)

File: SPECS/python-wcwidth/python-wcwidth.spec (hunk 1 above)
Comment (nicogbg) [reply SPECS/python-wcwidth/python-wcwidth.spec:48]: fixed

**Pull request merged**